guides/quick-start/bentoml-quick-start-guide.ipynb

```diff
@@ -156,6 +166,7 @@
     }
    ],
    "source": [
+    "%%writefile bento_packer.py\n",
```

Comment:
this part can not be just "%%writefile" in the notebook, otherwise you will need to import it again

Reply:
noted.